container/fib/README.md

```diff
@@ -2,41 +2,41 @@
 
 This package implements the **Forwarding Information Base (FIB)**.
 
-FIB is partitioned according to the [NDT](../ndt/).
+The FIB is partitioned according to the [NDT](../ndt).
 Most FIB entries only appear in one partition, as chosen by NDT lookup.
-In case a FIB entry name is shorter than NDT's prefix length, the FIB entry is duplicated across all partitions.
-Go `Fib` type represents the entire FIB; C `Fib` struct represents a single partition.
+In case a FIB entry name is shorter than the NDT's prefix length, the FIB entry will be replicated across all partitions.
+The `Fib` type in Go represents the entire FIB, while the `Fib` C struct represents a single partition.
 
-The FIB implements [2-stage LPM](http://ieeexplore.ieee.org/document/6665203/) algorithm for efficient Longest Prefix Match (LPM) lookups.
+The FIB implements a [2-stage LPM](https://doi.org/10.1109/ANCS.2013.6665203) algorithm for efficient **Longest Prefix Match (LPM)** lookups.
 
-## Go code
+## Go Code
 
-`Fib` type provides methods to execute read or update commands, sequentially executed in an RCU read-side thread that internally obtains and releases RCU read lock.
+The `Fib` type provides methods to execute read or update commands, sequentially executed in an RCU read-side thread that internally obtains and releases the RCU read lock. (FIXME: read side?)
```

Comment:
is this correct? the FIB is updated from a *read* side thread?

Reply:
This is indeed correct. See https://github.com/urcu/userspace-rcu/blob/4af3e30942f82d6c4b408ecac178e0beb50a4943/include/urcu/rculfhash.h#L347